compositions/aws-provider/example-application/definition.yaml

```diff
@@ -4,15 +4,15 @@
 apiVersion: apiextensions.crossplane.io/v1
 kind: CompositeResourceDefinition
 metadata:
-  name: xexampleapps.awsblueprints.io
+  name: xdynamodbexamples.awsblueprints.io
```

Comment:
I would rather not put the term DynamoDB in XRD. Reason is that this should define example applications in general, not tied to a specific type of application.